boxsdk/object/folder.py

```diff
@@ -142,6 +143,52 @@ def create_upload_session(self, file_size, file_name):
             response_object=response,
         )
 
+    def chunked_upload_with_stream(self, content_stream, file_size, file_name):
+        """
+        Instantiate the chunked upload instance and create upload session.
+
+        :param content_stream:
+            File-like object containing the content of the part to be uploaded.
+        :type content_stream:
+            :class:`File`
+        :param file_size:
+            The size of the file that this part belongs to.
+        :type file_size:
+            `int`
+        :param file_name:
+            The optional new name of the file
+        :type file_name:
+            `unicode` or None
+        :returns:
+            A :class:`ChunkedUpload` object.
+        :rtype:
+            :class:`ChunkedUpload`
+        """
+        upload_session = self.create_upload_session(file_size, file_name)
+        return ChunkedUpload(upload_session, content_stream, file_size)
+
+    def chunked_upload_with_path(self, file_path):
```

Comment:
This should have the `@api_call` decorator, since it ultimately makes an API call.  It might also be good to give this a name with a verb, e.g. `get_chunked_uploader()` or something